src/main/java/org/sunbird/common/util/ProjectUtil.java

```diff
@@ -202,4 +202,12 @@ public static Boolean validateDate(String dateString){
 	public static Boolean validateEmployeeId(String employeeId) {
 		return employeeId.matches("^(?=.*\\d|[a-zA-Z]{30})[a-zA-Z0-9 .-]{1,30}$"); // Allow only alphanumeric, numeric and restrict if only alphabets character
 	}
+
+	public static Boolean validateRegexPatternWithNoSpecialCharacter(String regex) {
+		return regex.matches("^[a-zA-Z0-9 -()]*$");
+	}
+
+	public static Boolean validatePinCode(String regex) {
+		return regex.matches("^[0-9]{4,7}$");
```

Comment:
why length is 4 to 7?
do we have pincode in this range?

Reply:
No @karthik-tarento  Updated to 6 digit standard used in india